scripts/composition.js

```diff
@@ -35,17 +35,21 @@
 
   const pixelSize = Hardware.Constants.PixelSize;
 
-  const boardWidth = HardwareProxy.Constants.Width;
-  const boardWidthInPixels = HardwareProxy.Constants.Width * pixelSize;
+  const physicalWidth = HardwareProxy.Constants.Width;
+  const physicalWidthInPixels = HardwareProxy.Constants.Width * pixelSize;
+  const physicalHeight = HardwareProxy.Constants.Height;
+
+  const boardWidth = 32;
+  const boardWidthInPixels = boardWidth * pixelSize;
+  const boardHeight = 32;
```

Comment:
I split the physical dimensions of the board (32x48) from the dimensions we need for the game (32x32).

In the current code, the board is like this:
```
BOARD1
2DRAOB
BOARD3
```
And we really use only board 1 and 2 for the game.

But maybe later we'll want to use the boards in the vertical direction instead (we could just rotate everything with CSS and keep the code ? not sure). Or put the game in the middle. I don't know.

Up to you actually :)